### PR TITLE
Update chart linting

### PR DESCRIPTION
**Pull request by KevinMGranger**
This is just https://github.com/konveyor/pelorus/pull/324 from my fork.

I'm getting a protected branch error that seems nonsensical.

```
remote: error: GH006: Protected branch update failed for refs/heads/update_chart_linting.
remote: error: At least 1 approving review is required by reviewers with write access.
To https://github.com/konveyor/pelorus.git
 ! [remote rejected] update_chart_linting -> update_chart_linting (protected branch hook declined)
error: failed to push some refs to 'https://github.com/konveyor/pelorus.git'
```

### CODE REVIEW

**Comment by KevinMGranger** [comment]
I updated my local helm version from 3.6.3 to 3.7.2 to match the one bundled from `ct`. I'm still seeing the dependency complaint locally but not in CI.

**Comment by KevinMGranger** [comment]
Possibly related to https://github.com/helm/chart-testing/issues/324.

**Comment by KevinMGranger** [comment]
I'm just going to update the linting tools and configuration, make it happy, and disregard the discrepancy for now. Confusing.

File: .gitignore [hunk 1]
```diff
@@ -146,4 +146,7 @@ policy/
 
 # BATS
 _test/test_helper/
-_test/prometheus/rules.yaml
\ No newline at end of file
+_test/prometheus/rules.yamlcharts/pelorus/Chart.lock
```
Comment (etsauer) [inline .gitignore:149]: this line doesn't look right.

**Review by etsauer — CHANGES_REQUESTED** [review]
one comment, otherwise looks good.

File: .gitignore (hunk 1 above)
Comment (KevinMGranger) [reply .gitignore:149]: Ah! I added the first path with VS code, which doesn't add line endings for the last line. But then the second path was `echo path >> .gitignore`, which meant it was on the same line. Treacherous. Will fix.

**Review by etsauer — APPROVED** [review]
lgtm

**Pull request merged**